bin/sozo/src/commands/build.rs

```diff
@@ -57,19 +70,50 @@ impl BuildArgs {
     }
 }
 
+pub fn print_stats(contract_statistic: ContractStatistics) {
+    println!(
+        "---------------Contract Stats for {}---------------",
+        contract_statistic.contract_name
+    );
+
+    println!(
+        "- Contract bytecode size (Number of felts in the program): {}\n",
+        contract_statistic.number_felts,
+    );
+
+    println!("- Contract Class size: {} bytes\n", contract_statistic.file_size);
```

Comment:
lets use `scarb`'s print methods instead of `println!` to be consistent with other parts of codebase, for example see: https://github.com/dojoengine/dojo/blob/364fe019a56d1e817fa80db2417bd09b768f7838/bin/sozo/src/commands/init.rs#L41-L42

Reply:
Will do.